src/main/features/claimant-response/views/defendants-response.njk

```diff
@@ -54,19 +55,55 @@
   </table>
 {% endmacro %}
 
+{% macro paymentInformationFragment(paymentDeclaration) %}
+  <h3 class="heading-small">{{ t('When they paid this amount') }}</h3>
+  <p>{{ claim.response.paymentDeclaration.paidDate | date }}</p>
```

Comment:
macro is accepting parameter `paymentDeclaration` while the `claim` object is being used here. please correct this

Reply:
try to extract these macros out of this file, to avoid these errors